### DISCUSSION TITLE
WIP - Warhammer 4e Layout

### DISCUSSION THREAD

**Discussion by Ascariel1984** [Obsidian-TTRPG-Community/fantasy-statblocks#322] [Show and tell]
Hi all, I am working on a Warhammer 4th edition layout for myself and wanted to share what I managed to do so far. Took me some time to figure out CSS and JS, but I am getting the hang of it I think. I would appreciate your input and suggestions and am happy to share the template once it's finished enough!

Currently I am unsure if I should add the hit locations with the armor points to the portrait. It looks a bit cleaner without, but adds information...
Also not sure about the Wounds, Fate/Fortune,... this would need either checkboxes or interactive + or - buttons to keep track.

![Paul Ech](https://github.com/javalent/fantasy-statblocks/assets/153175317/164001f7-6a30-4f6d-bee2-215ed5850eea)

**Comment by GabrielMPhi** [comment]
I was looking for that. Can I download it somewhere? I would save me so much time. It's great work!

**Comment by GabrielMPhi** [comment]
I can't wait :)

**Comment by marauder13** [comment]
Just adding my own call to get a copy of this. It looks great.

**Comment by Ascariel1984** [comment]
To be honest, I am not sure if this will be completed anytime soon. I had almost all formating and stuff ready, then I updated to the newest version and now everything is broken...
